src/lib/widgets/episode/ui/episode-card.svelte

```diff
@@ -1,17 +1,28 @@
 <script lang="ts">
+  import type { Episode } from '$lib/shared/api';
   import { PlaybackButton } from '$lib/features/playback-controls';
   import { LikeButton } from '$lib/features/like-episode';
   import { DownloadLink } from '$lib/features/download-episode';
   import { EpisodeCardShell } from '$lib/entities/episode';
-  import type { Episode } from '$lib/shared/api';
+  import { paused } from '$lib/entities/audio';
+  import { addToListeningHistory } from '$lib/entities/audio/model/listening-history';
```

Comment:
issue: according to FSD, you cannot go inside a slice from another slice, you must stop at the `index.ts`:
```suggestion
  import { addToListeningHistory } from '$lib/entities/audio';
```